### PR TITLE
Added allocated resource on booking page

### PR DESCRIPTION
**Pull request by Trolzie**
## Added Allocated Resource on Booking Page
We wanted to show the resource name while booking. This is to help people better understand who they are booking with.

## Design

screen shot 2017-10-03 at 16 29 23
screen shot 2017-10-03 at 16 29 32

## Other
This update shines light on an issue with backwards compatibility, and how the widget is slowly lagging behind the API endpoints. We need to soon make an effort to get this widget up to date!

## Branch name(s)

API: `api/team-booking-selected-resource`
Admin: `admin/team-booking-selected-resource`

## Who should review
@vistik

### CODE REVIEW

File: examples/local-strings.htm [hunk 1]
```diff
@@ -22,6 +22,7 @@
         app:      'back-to-the-future',
         localization: {
           strings: {
+            allocatedResourcePrefix: 'Your Guide',
```
Comment (vistik) [inline examples/local-strings.htm]: Your guide? is that the default?

Comment (vistik) [reply examples/local-strings.htm]: How will this look for someone that does not change anything  after we deploy?

Comment (Trolzie) [reply examples/local-strings.htm]: no, the default is in the 'defaults' file. the default being "with". This is just an example, and that is why I have chosen to change the text to something different.

Comment (vistik) [reply examples/local-strings.htm]: makes semse

**Comment by vistik** [comment]
🎱  looks ok to me

**Pull request merged**